frappe/public/js/frappe/widgets/widget_dialog.js

```diff
@@ -150,7 +150,13 @@ class ShortcutDialog extends WidgetDialog {
 				onchange: () => {
 					if (this.dialog.get_value("type") == "DocType") {
 						this.dialog.fields_dict.link_to.get_query = () => {
-							return { filters: { istable: false } };
+							return {
+								query: "frappe.core.report.permitted_documents_for_user.permitted_documents_for_user.query_doctypes",
+								filters: {
+									istable: false,
```

Comment:
`istable: false` is not used in `query_doctypes`. You may need to update `query_doctypes` method if you need this filter.

Reply:
Ohh yeah, just saw that.. Updating

Reply:
@surajshetty3416 turns out that the `build_permissions` functions excludes child tables by default